src/app/(marketing)/pricing/components/ClerkPricingTable.module.css

```
@@ -0,0 +1,432 @@
+}
+
+.cards :global(.cl-pricingTableCard:hover) {
+  --card-lift: -0.375rem;
+
+  z-index: 2;
+  box-shadow:
+    0 28px 58px -30px color-mix(in oklab, var(--foreground) 58%, transparent),
+    0 0 0 1px color-mix(in oklab, var(--primary) 18%, transparent);
+}
+
+@keyframes cardArrive {
```

Stylelint (17.14.1)

[error] Expected keyframe name "cardArrive" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)